kytos/core/interface.py

```diff
@@ -468,6 +740,8 @@ def as_dict(self):
             'status': self.status.value,
             'status_reason': sorted(self.status_reason),
             'link': self.link.id if self.link else "",
+            'available_tags': self.available_tags,
```

Comment:
Check out [my review on your topology PR](https://github.com/kytos-ng/topology/pull/168#discussion_r1348909103), even though this makes a lot of sense, I've asked a change here to try to avoid including on `as_dict()`, Thanks, Aldo.

Reply:
Got it, updated `ui` and now this PR is no longer needed. Closing it.